oqs/wrapper.py

```diff
@@ -361,5 +362,59 @@ def get_enabled_sig_mechanisms():
 
 
 def get_supported_sig_mechanisms():
-    """Returns list of supported signature mechanisms."""
+    """Returns the list of supported signature mechanisms."""
     return _supported_sigs
+
+
+# libOQS <oqs/rand.h> RNG support
+
+def randombytes(bytes_to_read):
```

Comment:
Why are we exposing the randombytes function?  I understand why we are exposing randombytes\_switch\_algorithm and nist\_kat\_init, but the actual randombytes function would only be used internally by implementations, external callers should use the appropriate Python routines for getting random bytes.

Reply:
@dstebila I exposed it in liboqs-cpp and liboqs-go, as it was marked as OQS_API in rand.h. I believe it's useful in testing (say e.g. we want a stream of random numbers from the nist rngs). My preference is to keep it, but if you have a strong opinion about it I can remove it.

Reply:
Something to discuss on the next dev meeting perhaps; we should make a decision and be consistent across all wrappers.

Reply:
@christianpaquin Good idea!

Reply:
> @dstebila I exposed it in liboqs-cpp and liboqs-go, as it was marked as OQS_API in rand.h. I believe it's useful in testing (say e.g. we want a stream of random numbers from the nist rngs). My preference is to keep it, but if you have a strong opinion about it I can remove it.

Do we use it in any tests?

Reply:
@dstebila Only in examples now, but I can imagine someone using it in testing.